docs/docs/README.md

```diff
@@ -15,9 +15,9 @@ At this stage the scope is providing the infrastructure for Bamboo DC.
 In order to deploy the infrastructure for Atlassian Data Center products on Kubernetes you need to have the 
```

Comment:
Is `infrastructure for Atlassian Data Center products on Kubernetes` the official title for the project? Just wondering if we can call it something a bit snappier?

Reply:
We should decide and unify the title, I see `infrastructure for Atlassian Data Center products on Kubernetes` and also`Terraform for Atlassian DC Products`.
We can simply called it data-center-terraform?

Reply:
@nghazali maybe we should pull Mike in on this?

Reply:
Right, we can run this with Mike and decide about the right name. Maybe just `Data Center Deployments` is sufficient as we gradually switch to terraform as the only official method for deployment so no need to repeat it.